pam/internal/adapter/newpasswordmodel.go

```diff
@@ -91,19 +93,28 @@ func (m newPasswordModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, nil
 
 		case "tab":
+			m.errorMsg = ""
+			if m.focusIndex == 0 && m.passwordEntries[0].Value() != "" {
+				err := checkChallengeQuality(m.currentChallenge, m.passwordEntries[0].Value())
+				if err != nil {
+					m.Clear()
+					return m, sendEvent(errMsgToDisplay{msg: err.Error()})
```

Comment:
As per said in a comment below, I think we should just show what `pwquality_strerror` gave to us here, and not the whole error string.